### PR TITLE
[Merged by Bors] - Bump post to v0.5.5

### PR DESCRIPTION
**Pull request by poszu**
## Motivation
Bumps post to use the new 8/56 proving algo

### CODE REVIEW

**Comment by poszu** [comment]
Bors try

**Review by fasmat — APPROVED** [review]

File: activation/post.go [hunk 1]
```diff
@@ -57,7 +57,7 @@ type PostProvingOpts struct {
 func DefaultPostProvingOpts() PostProvingOpts {
 	return PostProvingOpts{
 		Threads: 1,
-		Nonces:  20,
+		Nonces:  16,
```
Comment (fasmat) [inline activation/post.go:60]: I'm failing to understand why this parameter (that can be arbitrarily chosen by the operator of a node) influences the values we use for K1, K2, K3. Maybe you can enlighten me? 🙂

Comment (pigmej) [reply activation/post.go:60]: the default is "expected nonces" / "sensible amount of nonces" Kx affects those expected nonces (or the other way around if you look from the other side).

**Comment by poszu** [comment]
Bors merge

**Comment by poszu** [comment]
Bors merge

**Comment by pigmej** [comment]
@brusherru that means that the tool to check the performance have to go with nonces in 16 increments, 16,32,48 and soo on.

cc @poszu

**Pull request closed without merging**